uuidfield/fields.py

```diff
@@ -6,6 +6,11 @@
     from django.utils.encoding import smart_unicode
 except ImportError:
     from django.utils.encoding import smart_text as smart_unicode
+try:
+    from django.utils import six
```

Comment:
according to 

https://docs.djangoproject.com/en/dev/topics/python3/

Writing compatible code is much easier if you target Python ≥ 2.6. Django 1.5 introduces compatibility tools such as django.utils.six, which is a customized version of the six module. For convenience, forwards-compatible aliases were introduced in Django 1.4.2. If your application takes advantage of these tools, it will require Django ≥ 1.4.2.